ionos_common/external-dns-ionos/README.md

```diff
@@ -0,0 +1,43 @@
+Setup of External DNS Ionos Webhook, according to the README.md at URL:
```

Comment:
I would move it to the doc folder and maybe put a ref to it in the generale README

Reply:
Done